src/CqlBuilderPanel/common/utils.tsx

```diff
@@ -0,0 +1,49 @@
+// import { formatExpressionName } from "../definitionsSection/definitionBuilder/DefinitionBuilder";
+
+export const formatExpressionName = (values) => {
+  return values?.type !== "Timing" && values?.type !== "Pre-Defined Functions"
+    ? values?.type === "Functions" || values?.type === "Fluent Functions"
+      ? values?.name?.replace(/([\w\s]+)\(\)/g, '"$1"()')
+      : values?.name.includes(".")
+      ? values?.name.replace(/(.*\.)(.*)/, '$1"$2"')
+      : `"${values?.name}"`
+    : values?.name;
+};
+// given
+export const getNewExpressionsAndLines = (
```

Comment:
Do you think we will have value adding a unit test suite for this Util class?

Reply:
Possibly if a lot more is added to it. Right now It's just covered in it's own describe in another suite taht uses it